main.rb

```diff
@@ -1 +1,3 @@
 puts "Hello World"
+
+puts "Goodbye World"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.